### DISCUSSION TITLE
Generation 3d AI-based model

### DISCUSSION THREAD

**Discussion by thanhdat77** [Genesis-Embodied-AI/genesis-world#269] [Q&A]
Hi Genesis Team,

Thank you so much for the incredible work on launching this project—it's amazing!

I was exploring the "Papers Behind Genesis" and noticed some sections related to generative AI. However, I couldn’t find any details about prompts specifically for 3D object model-based generation. Could you share more about that?

Looking forward to learning more!

**Comment by zhouxian** [comment]
We either retrieve from existing 3D cad model database or resort to upstream 3D asset generation model, so this is not something we are spending effort on (yet) :)
